docs/source/development_faq.rst

```diff
@@ -1,19 +0,0 @@
-.. _development_faq:
```

Comment:
> Although your suggestion is reasonable [...]

Thanks for acknowledging that.

I restored the FAQ page, removing the second question: 6da2ea0cd123f874a359439cc8facb231305fac9